src/cryptoadvance/specter/templates/includes/coin_selection.html

```diff
@@ -0,0 +1,201 @@
+<template id="coin-selection">
+    <span class="toggle_coinselection" style="cursor: pointer;">{{ _("Coin selection") }} {% if show_advanced_settings %}&#9660;{% else %}&#9654;{% endif %}</span>
+    <br>
+    <div class="coinselection_div" style="margin: auto; max-width: 90%; display: none;">
+        <div class="errorMessagesDiv">
+            <message-box type="error" style="display: none;"></message-box>
+        </div>
+        <tx-table
+            class="txtable"
+            btc-unit="{{ specter.unit }}"
+            hide-sensitive-info="{{ specter.hide_sensitive_info | lower }}"
+            type="utxo" hide-columns="category time txid confirmations action" 
+            blockhash="true" hide-switcher="true">
+        </tx-table>
+    </div> 
+</template>
+
+
+<script type="module">
+    /**
+     * Custom element for showing the coin-selection-dialog
+       attributes:
```

Comment:
Minor thing but aren't `is-liquid` and `selected-coins` also attributes of the web component?